{"name": "Негативная проверка создания 2-ух одинаковых курьеров", "status": "passed", "description": "Запрос должен вернуть ошибку и правильный код ответа", "attachments": [{"name": "stdout", "source": "c66228fa-f0ba-4213-9df0-61c4300d0fa4-attachment.txt", "type": "text/plain"}], "start": 1721557355626, "stop": 1721557356168, "uuid": "973354c8-d34e-447f-bca2-1af5392f2943", "historyId": "a1c163e6329341bce654e28c1cebd101", "testCaseId": "a1c163e6329341bce654e28c1cebd101", "fullName": "tests_create_courier.test_creating_2_identical_couriers.TestCreate2IdenticalCouriers#test_сreate_2_identical_couriers_error", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('reg')"}, {"name": "parentSuite", "value": "tests_create_courier"}, {"name": "suite", "value": "test_creating_2_identical_couriers"}, {"name": "subSuite", "value": "TestCreate2IdenticalCouriers"}, {"name": "host", "value": "DESKTOP-IKUPUFD"}, {"name": "thread", "value": "4440-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests_create_courier.test_creating_2_identical_couriers"}]}